sw/device/examples/sram_program/BUILD

```diff
@@ -31,6 +32,12 @@ opentitan_ram_binary(
         "//sw/device/lib/base:macros",
         "//sw/device/lib/dif:sram_ctrl",
         "//sw/device/lib/runtime:log",
+        "//sw/device/lib/testing:pinmux_testutils",
         "//sw/device/lib/testing/test_framework:check",
     ],
 )
+
+opentitan_sram_fpga_cw310_test(
+    name = "sram_program_fpga_cw310_test",
+    sram_program_elf = ":sram_program_fpga_cw310.elf",
```

Comment:
I suppose this might need `tags = ["manual"]` depending on the CI hardware configuration.

Reply:
Just did this.